src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.Browser.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Win32.SafeHandles;
+
+using System.ComponentModel;
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Runtime.Versioning;
+using System.Threading;
+
+namespace System.Net.NetworkInformation
+{
+    public partial class NetworkChange
```

Comment:
Why not do this like other platform specific partial classes like `NetworkAddressChange.Unix.cs`?

Reply:
I don't understand what you mean. Can you please give me a clue?